.circleci/config.yml

```diff
@@ -104,21 +130,44 @@ commands:
       skipfile:
         type: string
         default: notafile
+      usegrackle:
+        type: boolean
+        default: true
+      skiptest:
+        type: boolean
+        default: false
     steps:
       - run:
-          name: "Compile enzo-e and run tests."
+          name: "Fetch enzo-e files enzo-e and run tests."
```

Comment:
Did you mean "Fetch enzo-e files and run tests" or something similar?

Reply:
Thanks for catching this. I think I changed my mind mid edit or something... 

I'm changing it to say, "Checkout target tag from enzo-e repository"